config/dev.json

```diff
@@ -11,6 +11,7 @@
   "jobManagerUrlRoot": "https://job-manager.dsde-dev.broadinstitute.org/jobs",
   "leoUrlRoot": "https://leonardo.dsde-dev.broadinstitute.org",
   "marthaUrlRoot": "https://us-central1-broad-dsde-dev.cloudfunctions.net",
+  "metadataEndpoint": "martha_v2",
```

Comment:
I'm curious why this should be a config option. Normally these are used for things that need to differ between environments, but these seem to all be the same. Is this being used like a feature flag, or for instrumentation? If that's the case, there are other mechanisms that might be a better fit for that.

Reply:
There are two different endpoints that provide metadata for DOS/DRS URIs, `martha_v2` and `martha_v3` (which supports the Terra Data Repo), and a separate endpoint that provides signed Urls (`getSignedUrlV1`).  We wanted to make it a configurable option so that we can roll back to `v2` if something is not working properly with `v3`, and the code needs to have a way for it to know which endpoint it's using because the responses come in different shapes.  I would be happy to implement this in another way; this was the cleanest I could come up with.

Reply:
It's so that we can switch out `martha_v2` for `martha_v3` with just a config change and no code

A separation of concerns sort of thing